### PR TITLE
fix encoding - part 2

### PR DESCRIPTION
**Pull request by sdwheeler**
This updates the encoding information per issue #4155. The changes are broken into 4 PRs for easier review.

### CODE REVIEW

**Comment by zjalexander** [comment]
Docs Build status updates of commit _[0e2fe89](https://github.com/sdwheeler/PowerShell-Docs/pull/4203/commits/0e2fe89401e3cffa8183eaabe16d33607d8ed162)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/3.0/Microsoft.PowerShell.Management/Get-Content.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue4155-b/reference/3.0/Microsoft.PowerShell.Management/Get-Content.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/get-content?view=powershell-3.0&branch=pr-en-us-4203) (powershell-3.0) |
[reference/3.0/Microsoft.PowerShell.Utility/Export-PSSession.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue4155-b/reference/3.0/Microsoft.PowerShell.Utility/Export-PSSession.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.utility/export-pssession?view=powershell-3.0&branch=pr-en-us-4203) (powershell-3.0) |
[reference/3.0/Microsoft.PowerShell.Utility/Import-Clixml.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue4155-b/reference/3.0/Microsoft.PowerShell.Utility/Import-Clixml.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.utility/import-clixml?view=powershell-3.0&branch=pr-en-us-4203) (powershell-3.0) |
[reference/4.0/Microsoft.PowerShell.Management/Get-Content.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue4155-b/reference/4.0/Microsoft.PowerShell.Management/Get-Content.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/get-content?view=powershell-4.0&branch=pr-en-us-4203) (powershell-4.0) |
[reference/4.0/Microsoft.PowerShell.Utility/Export-PSSession.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue4155-b/reference/4.0/Microsoft.PowerShell.Utility/Export-PSSession.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.utility/export-pssession?view=powershell-4.0&branch=pr-en-us-4203) (powershell-4.0) |
[reference/4.0/Microsoft.PowerShell.Utility/Import-Clixml.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue4155-b/reference/4.0/Microsoft.PowerShell.Utility/Import-Clixml.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.utility/import-clixml?view=powershell-4.0&branch=pr-en-us-4203) (powershell-4.0) |
[reference/5.0/Microsoft.PowerShell.Management/Get-Content.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue4155-b/reference/5.0/Microsoft.PowerShell.Management/Get-Content.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/get-content?view=powershell-5.0&branch=pr-en-us-4203) (powershell-5.0) |
[reference/5.0/Microsoft.PowerShell.Utility/Export-PSSession.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue4155-b/reference/5.0/Microsoft.PowerShell.Utility/Export-PSSession.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.utility/export-pssession?view=powershell-5.0&branch=pr-en-us-4203) (powershell-5.0) |
[reference/5.0/Microsoft.PowerShell.Utility/Format-Hex.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue4155-b/reference/5.0/Microsoft.PowerShell.Utility/Format-Hex.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.utility/format-hex?view=powershell-5.0&branch=pr-en-us-4203) (powershell-5.0) |
[reference/5.0/Microsoft.PowerShell.Utility/Import-Clixml.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue4155-b/reference/5.0/Microsoft.PowerShell.Utility/Import-Clixml.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.utility/import-clixml?view=powershell-5.0&branch=pr-en-us-4203) (powershell-5.0) |
[reference/5.1/Microsoft.PowerShell.Management/Get-Content.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue4155-b/reference/5.1/Microsoft.PowerShell.Management/Get-Content.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/get-content?view=powershell-5.1&branch=pr-en-us-4203) (powershell-5.1) |
[reference/5.1/Microsoft.PowerShell.Utility/Export-PSSession.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue4155-b/reference/5.1/Microsoft.PowerShell.Utility/Export-PSSession.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.utility/export-pssession?view=powershell-5.1&branch=pr-en-us-4203) (powershell-5.1) |
[reference/5.1/Microsoft.PowerShell.Utility/Format-Hex.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue4155-b/reference/5.1/Microsoft.PowerShell.Utility/Format-Hex.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.utility/format-hex?view=powershell-5.1&branch=pr-en-us-4203) (powershell-5.1) |
[reference/5.1/Microsoft.PowerShell.Utility/Import-Clixml.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue4155-b/reference/5.1/Microsoft.PowerShell.Utility/Import-Clixml.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.utility/import-clixml?view=powershell-5.1&branch=pr-en-us-4203) (powershell-5.1) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C4%5C23%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201904231842480912-4203%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=F24cIdaLtq87LwX3q5Qfi8diTbRx0kD3130%2BoAb8MH4%3D&st=2019-04-23T18%3A42%3A40Z&se=2019-05-24T18%3A47%3A40Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: reference/3.0/Microsoft.PowerShell.Utility/Export-PSSession.md [hunk 1]
```diff
@@ -253,19 +253,15 @@ Accept wildcard characters: False
 
 ### -Encoding
 
-Specifies the type of encoding for the target file. The default value is **ASCII**.
+Specifies the type of encoding for the target file. The default value is **UTF8**.
```
Comment (DCtheGeek) [inline reference/3.0/Microsoft.PowerShell.Utility/Export-PSSession.md]: Ensure acceptable values match in YAML to new list.

File: reference/5.0/Microsoft.PowerShell.Utility/Format-Hex.md [hunk 2]
```diff
@@ -90,19 +91,13 @@ extension **.t7f** is uncommon, but the hexadecimal output **%PDF** shows that i
 
 ### -Encoding
 
-Specifies the type of encoding for the target file. The default value is **ASCII**.
+Specifies the type of encoding for the target file. The default value is **Default**.
```
Comment (DCtheGeek) [inline reference/5.0/Microsoft.PowerShell.Utility/Format-Hex.md]: Ensure acceptable values match in YAML to new list and update default value.

**Comment by zjalexander** [comment]
Docs Build status updates of commit _[14e11dc](https://github.com/sdwheeler/PowerShell-Docs/pull/4203/commits/14e11dcb550657a14f1b10c638cd7da0483e10e1)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/3.0/Microsoft.PowerShell.Management/Get-Content.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue4155-b/reference/3.0/Microsoft.PowerShell.Management/Get-Content.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/get-content?view=powershell-3.0&branch=pr-en-us-4203) (powershell-3.0) |
[reference/3.0/Microsoft.PowerShell.Utility/Export-PSSession.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue4155-b/reference/3.0/Microsoft.PowerShell.Utility/Export-PSSession.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.utility/export-pssession?view=powershell-3.0&branch=pr-en-us-4203) (powershell-3.0) |
[reference/3.0/Microsoft.PowerShell.Utility/Import-Clixml.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue4155-b/reference/3.0/Microsoft.PowerShell.Utility/Import-Clixml.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.utility/import-clixml?view=powershell-3.0&branch=pr-en-us-4203) (powershell-3.0) |
[reference/4.0/Microsoft.PowerShell.Management/Get-Content.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue4155-b/reference/4.0/Microsoft.PowerShell.Management/Get-Content.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/get-content?view=powershell-4.0&branch=pr-en-us-4203) (powershell-4.0) |
[reference/4.0/Microsoft.PowerShell.Utility/Export-PSSession.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue4155-b/reference/4.0/Microsoft.PowerShell.Utility/Export-PSSession.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.utility/export-pssession?view=powershell-4.0&branch=pr-en-us-4203) (powershell-4.0) |
[reference/4.0/Microsoft.PowerShell.Utility/Import-Clixml.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue4155-b/reference/4.0/Microsoft.PowerShell.Utility/Import-Clixml.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.utility/import-clixml?view=powershell-4.0&branch=pr-en-us-4203) (powershell-4.0) |
[reference/5.0/Microsoft.PowerShell.Management/Get-Content.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue4155-b/reference/5.0/Microsoft.PowerShell.Management/Get-Content.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/get-content?view=powershell-5.0&branch=pr-en-us-4203) (powershell-5.0) |
[reference/5.0/Microsoft.PowerShell.Utility/Export-PSSession.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue4155-b/reference/5.0/Microsoft.PowerShell.Utility/Export-PSSession.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.utility/export-pssession?view=powershell-5.0&branch=pr-en-us-4203) (powershell-5.0) |
[reference/5.0/Microsoft.PowerShell.Utility/Format-Hex.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue4155-b/reference/5.0/Microsoft.PowerShell.Utility/Format-Hex.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.utility/format-hex?view=powershell-5.0&branch=pr-en-us-4203) (powershell-5.0) |
[reference/5.0/Microsoft.PowerShell.Utility/Import-Clixml.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue4155-b/reference/5.0/Microsoft.PowerShell.Utility/Import-Clixml.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.utility/import-clixml?view=powershell-5.0&branch=pr-en-us-4203) (powershell-5.0) |
[reference/5.1/Microsoft.PowerShell.Management/Get-Content.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue4155-b/reference/5.1/Microsoft.PowerShell.Management/Get-Content.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/get-content?view=powershell-5.1&branch=pr-en-us-4203) (powershell-5.1) |
[reference/5.1/Microsoft.PowerShell.Utility/Export-PSSession.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue4155-b/reference/5.1/Microsoft.PowerShell.Utility/Export-PSSession.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.utility/export-pssession?view=powershell-5.1&branch=pr-en-us-4203) (powershell-5.1) |
[reference/5.1/Microsoft.PowerShell.Utility/Format-Hex.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue4155-b/reference/5.1/Microsoft.PowerShell.Utility/Format-Hex.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.utility/format-hex?view=powershell-5.1&branch=pr-en-us-4203) (powershell-5.1) |
[reference/5.1/Microsoft.PowerShell.Utility/Import-Clixml.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue4155-b/reference/5.1/Microsoft.PowerShell.Utility/Import-Clixml.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.utility/import-clixml?view=powershell-5.1&branch=pr-en-us-4203) (powershell-5.1) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C4%5C23%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201904232108591273-4203%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=jtSw1GZxYgquEIwsm4Z0%2Fyk7QqnSXLy7wzh6TuB4GYA%3D&st=2019-04-23T21%3A08%3A48Z&se=2019-05-24T21%3A13%3A48Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: reference/5.0/Microsoft.PowerShell.Utility/Format-Hex.md (hunk 2 above)
Comment (sdwheeler) [reply reference/5.0/Microsoft.PowerShell.Utility/Format-Hex.md]: fixed

File: reference/3.0/Microsoft.PowerShell.Utility/Export-PSSession.md (hunk 1 above)
Comment (sdwheeler) [reply reference/3.0/Microsoft.PowerShell.Utility/Export-PSSession.md]: fixed

**Pull request merged**